pkg/functions/builder_test.go

```diff
@@ -128,3 +91,23 @@ func errorMock(statusCode int, message string) func(req *http.Request) (*http.Re
 		}, nil
 	}
 }
+
+func Test_copyFunctionTemplate(t *testing.T) {
```

Comment:
We should really be consistent about our test naming.  Seems that you prefer the Test_ nomenclature, but everyone else is just using camel case.  We need to pick one.

Reply:
👍 for consistency.

If there you want to write unit tests for `FooBar` and `fooBar` functions, how would you suggest tests be named? I'd name them `TestFooBar` and `Test_fooBar`.